packages/grid/x-data-grid/src/hooks/features/columnHeaders/useGridColumnHeaders.tsx

```diff
@@ -407,6 +407,10 @@ export const useGridColumnHeaders = (props: UseGridColumnHeadersProps) => {
         })
         .filter((groupStructure) => groupStructure.columnFields.length > 0);
 
+      if (!visibleColumnGroupHeader.length) {
+        return null;
+      }
+
```

Comment:
This return is inside a loop that iterates through all header groups. Is it really correct to skip all other header groups? Shouldn't this be a `continue`?

And if it's correct (e.g. because one group with no visible columns implies no group has visible columns), then could we instead abort before starting the loop?

Reply:
i agree you. i changed `return` to `continue`

Reply:
this eslint has no-continue rule. what should i do?